src/rgw/driver/rados/rgw_data_sync.cc

```diff
@@ -3105,12 +3170,24 @@ void RGWRemoteDataLog::wakeup(int shard_id, bc::flat_set<rgw_data_notify_entry>&
 
 int RGWRemoteDataLog::run_sync(const DoutPrefixProvider *dpp, int num_shards)
 {
+  // construct and start bid manager for data sync fairness
+  const auto& control_pool = sc.env->driver->svc()->zone->get_zone_params().control_pool;
+  auto control_obj = rgw_raw_obj{control_pool, data_sync_bids_oid};
```

Comment:
unlike meta sync where we sync from a single master zone, data sync will run against each source zone independently. so i think we'll need to use a different rados object for each source zone. this will matter for zonegroups with more than two zones

for the data sync status objects, we include the source zone id in the object name - see `RGWDataSyncStatusManager::sync_status_oid()`